src/Google.Api.Gax/Api/Gax/GaxPreconditions.cs

```diff
@@ -6,6 +6,7 @@
  */
 
 using System;
+using Google.Protobuf;
```

Comment:
Why has this been added? There don't appear to be any other changes in this file?